modules/ROOT/pages/provisioning-virtualbox.adoc

```diff
@@ -0,0 +1,195 @@
+= Provisioning Fedora CoreOS on VirtualBox
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on the VirtualBox hypervisor.
+
+== Prerequisites
+
+Before provisioning an FCOS machine, you must have an Ignition configuration file containing your customizations and host it somewhere (e.g. using `python3 -m http.server`, we will assume this option is used in the following examples). If you do not alrady have an Ignition file, see xref:producing-ign.adoc[Producing an Ignition File].
+
+== Setting up a new VM
+
+You can set up a VirtualBox based Virtual Machine through the GUI or via the https://www.virtualbox.org/manual/UserManual.html#vboxmanage[`vboxmanage` CLI]. Below is a shell script that does most of what the graphical wizard will do. Note that it is tailored to the installation from live ISO or Raw disk image as outlined in the following sections.
+
+NOTE: Please be aware that for now there is no dedicated Fedora CoreOS image available for VirtualBox and the installation instructions provided are essentially xref:bare-metal.adoc[bare metal instructions]. A dedicated image may be provided at a later time. Please check the GitHub issues https://github.com/coreos/fedora-coreos-tracker/issues/73[#73] and https://github.com/coreos/fedora-coreos-tracker/issues/144[#144] for more details.
+
+We will rely on the VirtualBox provided NAT gateway to access the hosts's loopback interface on `10.0.2.2`. Check the https://www.virtualbox.org/manual/UserManual.html#network_nat[VirtualBox Documentation] for more details.
+
+[source, bash]
+----
+VM_NAME='fcos-node01'
+VM_DISK=$(realpath ~/VirtualBox\ VMs/${VM_NAME})/${VM_NAME}.vdi
+
+vboxmanage createvm --name "${VM_NAME}" --ostype Fedora_64 --register
+# Set hardware settings for the VM, including recommended graphics controller
+vboxmanage modifyvm "${VM_NAME}" --memory 4096 --cpus 2 --vram 20 --graphicscontroller vmsvga --rtcuseutc on --usb on --usbehci on
```

Comment:
Is USB actually required?

Reply:
Fair question: I doubt it but as described in the leading paragraph, the shell command(s) are designed to give a very similar output the GUI wizard. I can test that again without USB and fix this later accordingly.